.travis.yml

```diff
@@ -24,9 +24,4 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - sleep 3
 script:
-  - npm run lint
-  - npm test
-  - npm run build:prod
-  - npm run server:prod:ci &
-  - sleep 3
-  - npm run e2e
+  - npm run ci
```

Comment:
I personally use Circle but this should still be applicable ..

- I agree completely in regards to testing both builds, the point i was trying to make is to separate the two completely to allow the parallel execution. I have multiple containers available for any given build so my Jit & AoT validations run in parallel. Given it's a devops thing and most people aren't going to be running the vanilla Travis file for their projects, no need to have the build config hold up this PR.

- As far as the e2e tests go, I run a `sleep 5` after I spin up the server. I've had problems with 2-3 second sleeps when it's build prime time on Travis.